src/cmap/commands.ts

```diff
@@ -429,10 +429,66 @@ export interface OpMsgOptions {
 
 /** @internal */
 export class DocumentSequence {
+  field: string;
   documents: Document[];
+  serializedDocumentsLength: number;
+  private chunks: Uint8Array[];
+  private header?: Buffer;
```

Comment:
```suggestion
  private header: Buffer;
```

header can't be null, right?

Reply:
Updated so it is never undefined.